pydy/tests/test_models.py

```diff
@@ -19,8 +16,8 @@ def test_multi_mass_spring_damper():
 
     sys = multi_mass_spring_damper()
 
-    assert Counter(sys.constants_symbols) == Counter([k0, c0, m0])
-    assert sys.specifieds_symbols == list()
+    assert sys.constants_symbols == {k0, c0, m0}
```

Comment:
The ordering differences are probably due to the fact that various methods in SymPy do not guarantee to spit out the same order of symbols when you search an expression tree. But our System class relies on keeping the order constant once those sequences arrive internally to System.

Which tests were failing wrt to the ordering? We've had some other issues like that too that depend on SymPy versions or even the machine that the the code was run on.